index.html

```diff
@@ -3,21 +3,14 @@
 <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta name="description" content=""Green Earth - Learn about the benefits of plants and shopfor eco-friendly greenery to enhance your home and health.">
+    <meta name="description" content="Green Earth - Learn about the benefits of plants and shopfor eco-friendly greenery to enhance your home and health.">
     <title>Green Earth - Home</title>
```

Comment:
You need to import the Materialize CSS CDN links here for the styles like your `class="video-container"` code for the YT video to work.

```
  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css">
   <script src="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/js/materialize.min.js"></script>
```